packages/@aws-cdk/aws-sns/test/test.sns.ts

```diff
@@ -103,6 +103,48 @@ export = {
 
       test.done();
     },
+
+    'specify contentBasedDeduplication'(test: Test) {
```

Comment:
With regards to comment on `contentBasedDeduplication`, this test should actually assert that an error is thrown since `fifo` is not enabled.